src/deriver/queue_payload.py

```
@@ -0,0 +1,92 @@
+            target_name: Name of the observer peer
+            task_type: Type of task ('representation' or 'summary')
+
+        Returns:
+            Processed payload dictionary ready for queue processing
+
+        Raises:
+            ValueError: If the payload doesn't match the expected schema
+        """
+        # Validate required fields and types
+        if not isinstance(message.get("content"), str):
+            raise TypeError("Message content must be a string")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)